### PR TITLE
feat: add edge.auto dedicated playbook

### PR DESCRIPTION
**Pull request by manato**
## Related Links

- https://github.com/tier4/edge-auto-jetson/pull/26

## Description

This PR adds a new playbook and roles for Edge.auto

## Review Procedure

```sh
setup-dev-env.sh
```

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [x] Commits are properly organized and messages are according to the guideline
- [ ] (Optional) Unit tests have been written for new behavior
- [x] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets

## CI Checks

- **Build and test for PR**: Required to pass before the merge.

### CODE REVIEW

**Comment by aohsato** [comment]
@manato How about PR status?

**Comment by manato** [comment]
@aohsato Because I guess all modifications have been done so far, I'll turn this PR to be open

File: ansible/playbooks/edge_auto.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    - role: autoware.dev_env.tensorrt
+      when: prompt_install_nvidia == 'y'
+    - role: autoware.dev_env.git_lfs
+
+    # Edge.Auto
+    - role: autoware.dev_env.ros2_domain_id
+    - role: autoware.dev_env.cyclonedds
+    - role: autoware.dev_env.ptp4l_host
+    - role: autoware.dev_env.enlarge_rx_ring_buffer
+    - role: autoware.dev_env.netplan
```
Comment (aohsato) [inline ansible/playbooks/edge_auto.yaml:60]: [imo] It's better that user can choose netplan installation or not.

Comment (aohsato) [reply ansible/playbooks/edge_auto.yaml:66]: nice, too!
https://github.com/tier4/edge-auto/pull/4/commits/8015a1a856afe1cf67ecc4dd04bc3035941dd3d8

**Review by aohsato — APPROVED** [review]
Working on my environment, also good work.

**Pull request merged**